keps/sig-cluster-lifecycle/kubeadm/20190506-kubeadm-machine-output.md

```diff
@@ -0,0 +1,360 @@
lt3YWl0LWNvbnRyb2wtcGxhbmVdIFdhaXRpbmcgZm9yIHRoZSBrdWJlbGV0IHRvIGJvb3QgdXAgdGhlIGNvbnRyb2wgcGxhbmUgYXMgc3RhdGljIFBvZHMgZnJvbSBkaXJlY3RvcnkgIi9ldGMva3ViZXJuZXRlcy9tYW5pZmVzdHMiLiBUaGlzIGNhbiB0YWtlIHVwIHRvIDRtMHMKW2FwaWNsaWVudF0gQWxsIGNvbnRyb2wgcGxhbmUgY29tcG9uZW50cyBhcmUgaGVhbHRoeSBhZnRlciAxNy4wMDE2ODggc2Vjb25kcwpbdXBsb2FkLWNvbmZpZ10gc3RvcmluZyB0aGUgY29uZmlndXJhdGlvbiB1c2VkIGluIENvbmZpZ01hcCAia3ViZWFkbS1jb25maWciIGluIHRoZSAia3ViZS1zeXN0ZW0iIE5hbWVzcGFjZQpba3ViZWxldF0gQ3JlYXRpbmcgYSBDb25maWdNYXAgImt1YmVsZXQtY29uZmlnLTEuMTQiIGluIG5hbWVzcGFjZSBrdWJlLXN5c3RlbSB3aXRoIHRoZSBjb25maWd1cmF0aW9uIGZvciB0aGUga3ViZWxldHMgaW4gdGhlIGNsdXN0ZXIKW3VwbG9hZC1jZXJ0c10gU2tpcHBpbmcgcGhhc2UuIFBsZWFzZSBzZWUgLS1leHBlcmltZW50YWwtdXBsb2FkLWNlcnRzClttYXJrLWNvbnRyb2wtcGxhbmVdIE1hcmtpbmcgdGhlIG5vZGUgYzAxLjBlYzlkMjEuc2s4IGFzIGNvbnRyb2wtcGxhbmUgYnkgYWRkaW5nIHRoZSBsYWJlbCAibm9kZS1yb2xlLmt1YmVybmV0ZXMuaW8vbWFzdGVyPScnIgpbbWFyay1jb250cm9sLXBsYW5lXSBNYXJraW5nIHRoZSBub2RlIGMwMS4wZWM5ZDIxLnNrOCBhcyBjb250cm9sLXBsYW5lIGJ5IGFkZGluZyB0aGUgdGFpbnRzIFtub2RlLXJvbGUua3ViZXJuZXRlcy5pby9tYXN0ZXI6Tm9TY2hlZHVsZV0KW2Jvb3RzdHJhcC10b2tlbl0gVXNpbmcgdG9rZW46IHNkaHdxMi4zaHFoNmg3dTA2ZGJzaGNkCltib290c3RyYXAtdG9rZW5dIENvbmZpZ3VyaW5nIGJvb3RzdHJhcCB0b2tlbnMsIGNsdXN0ZXItaW5mbyBDb25maWdNYXAsIFJCQUMgUm9sZXMKW2Jvb3RzdHJhcC10b2tlbl0gY29uZmlndXJlZCBSQkFDIHJ1bGVzIHRvIGFsbG93IE5vZGUgQm9vdHN0cmFwIHRva2VucyB0byBwb3N0IENTUnMgaW4gb3JkZXIgZm9yIG5vZGVzIHRvIGdldCBsb25nIHRlcm0gY2VydGlmaWNhdGUgY3JlZGVudGlhbHMKW2Jvb3RzdHJhcC10b2tlbl0gY29uZmlndXJlZCBSQkFDIHJ1bGVzIHRvIGFsbG93IHRoZSBjc3JhcHByb3ZlciBjb250cm9sbGVyIGF1dG9tYXRpY2FsbHkgYXBwcm92ZSBDU1JzIGZyb20gYSBOb2RlIEJvb3RzdHJhcCBUb2tlbgpbYm9vdHN0cmFwLXRva2VuXSBjb25maWd1cmVkIFJCQUMgcnVsZXMgdG8gYWxsb3cgY2VydGlmaWNhdGUgcm90YXRpb24gZm9yIGFsbCBub2RlIGNsaWVudCBjZXJ0aWZpY2F0ZXMgaW4gdGhlIGNsdXN0ZXIKW2Jvb3RzdHJhcC10b2tlbl0gY3JlYXRpbmcgdGhlICJjbHVzdGVyLWluZm8iIENvbmZpZ01hcCBpbiB0aGUgImt1YmUtcHVibGljIiBuYW1lc3BhY2UKW2FkZG9uc10gQXBwbGllZCBlc3NlbnRpYWwgYWRkb246IENvcmVETlMKW2FkZG9uc10gQXBwbGllZCBlc3NlbnRpYWwgYWRkb246IGt1YmUtcHJveHkKCllvdXIgS3ViZXJuZXRlcyBjb250cm9sLXBsYW5lIGhhcyBpbml0aWFsaXplZCBzdWNjZXNzZnVsbHkhCgpUbyBzdGFydCB1c2luZyB5b3VyIGNsdXN0ZXIsIHlvdSBuZWVkIHRvIHJ1biB0aGUgZm9sbG93aW5nIGFzIGEgcmVndWxhciB1c2VyOgoKICBta2RpciAtcCAkSE9NRS8ua3ViZQogIHN1ZG8gY3AgLWkgL2V0Yy9rdWJlcm5ldGVzL2FkbWluLmNvbmYgJEhPTUUvLmt1YmUvY29uZmlnCiAgc3VkbyBjaG93biAkKGlkIC11KTokKGlkIC1nKSAkSE9NRS8ua3ViZS9jb25maWcKCllvdSBzaG91bGQgbm93IGRlcGxveSBhIHBvZCBuZXR3b3JrIHRvIHRoZSBjbHVzdGVyLgpSdW4gImt1YmVjdGwgYXBwbHkgLWYgW3BvZG5ldHdvcmtdLnlhbWwiIHdpdGggb25lIG9mIHRoZSBvcHRpb25zIGxpc3RlZCBhdDoKICBodHRwczovL2t1YmVybmV0ZXMuaW8vZG9jcy9jb25jZXB0cy9jbHVzdGVyLWFkbWluaXN0cmF0aW9uL2FkZG9ucy8KClRoZW4geW91IGNhbiBqb2luIGFueSBudW1iZXIgb2Ygd29ya2VyIG5vZGVzIGJ5IHJ1bm5pbmcgdGhlIGZvbGxvd2luZyBvbiBlYWNoIGFzIHJvb3Q6CgprdWJlYWRtIGpvaW4gMTkyLjE2OC4yMC41MTo0NDMgLS10b2tlbiBzZGh3cTIuM2hxaDZoN3UwNmRic2hjZCBcCiAgICAtLWRpc2NvdmVyeS10b2tlbi1jYS1jZXJ0LWhhc2ggc2hhMjU2OjFmNDBmZjRiZDFiODU0ZmI0YTVjZjVkMmYzODI2N2E1Y2U1Zjg5ZTM0ZDM0YjBmNjJiZjMzNWQ3NGVlZjkxYTMgCg=="
```

Comment:
Shorten this line for the readability of the md.

Reply:
Hi @randomvariable,

I'm not sure I agree. I want someone to be able to decode the data and see that it's actual output from `kubeadm init`. When rendered as Markdown the line is a single line, and does not interfere with reviews.

Reply:
Also, the length is why I added it the "appendix"

Tell you what, I am going to replace it with text like:

```shell
For the actual output please see this Gist URL
```

Reply:
Fixed